src/GT.UI/Controllers/ListingsController.cs

```diff
@@ -1,118 +1,97 @@
 ﻿#nullable disable
-using GT.Data.Data.GTAppDb;
-using GT.Data.Data.GTAppDb.Entities;
+
+using GT.Core.DTO.Impl;
+using GT.Core.FilterModels.Interfaces;
+using GT.Core.Services.Interfaces;
+using GT.Data.Data.GTIdentityDb.Entities;
+using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.EntityFrameworkCore;
 
 namespace GT.UI.Controllers
 {
 	[Route("api/[controller]")]
 	[ApiController]
 	public class ListingsController : ControllerBase
 	{
-		private readonly GTAppContext _context;
+		private readonly IGTListingService _listingService;
+		private readonly UserManager<ApplicationUser> _userManager;
 
-		public ListingsController(GTAppContext context)
+		public ListingsController(IGTListingService listingService, UserManager<ApplicationUser> userManager)
 		{
-			_context = context;
+			_listingService = listingService;
+			_userManager = userManager;
 		}
 
 		// GET: api/Listings
 		[HttpGet]
-		public async Task<ActionResult<IEnumerable<Listing>>> GetListings()
+		public async Task<ActionResult<IEnumerable<ListingDTO>>> GetListings(IListingFilterModel filterModel = null)
```

Comment:
This will return a `Task<ActionResult<IEnumerable<ListingPartialDTO>>>` (which will be changed to ListingOverviewDTO in PR #79).

Reply:
Fixed in commit